newsroom/monitoring/email_alerts.py

```diff
@@ -239,6 +239,16 @@ def already_sent(self, item, profile):
 
         return get_resource_service('history').get(req=None, lookup=lookup).count()
 
+    def validate_users(self, m, company):
```

Comment:
The name is a bit misleading, as it's actually filtering and not validating the supplied args